moveit_simple/src/moveit_simple.cpp

```diff
@@ -478,6 +595,23 @@ Eigen::Affine3d Robot::transformToBase(const Eigen::Affine3d &in,
 
 
 
+bool Robot::getFK(const std::vector<double> & joint_point,
+                  Eigen::Affine3d &pose) const
+{
+  robot_state_->setJointGroupPositions(joint_group_, joint_point);
+  const std::vector<std::string> link_names = robot_model_ptr_->getLinkModelNames();
+  if (!link_names.empty())
```

Comment:
@shaun-edwards ! Is it fine to include a fixed joint in the chain as done in the [kuka resource file](https://github.com/shaun-edwards/moveit_simple/blob/indigo-devel/moveit_simple/test/resources/kuka_kr210/kuka_kr210.srdf#L13). Because if it is then the condition you have mentioned will fail. In this joint model, there is one extra joint ([link_6-tool0](https://github.com/shaun-edwards/moveit_simple/blob/indigo-devel/moveit_simple/test/resources/kuka_kr210/kr210l150.urdf#L95)) apart from 6 active joints